README.md

```diff
@@ -1,12 +1,13 @@
 # Feature Hub
 
-[![Build Status][travis-ci-badge]][travis-ci]
-[![Website][website-badge]][website]
-
 The Feature Hub is an opinionated JavaScript implementation of the micro
 frontends approach to creating scalable web applications with multiple teams and
 different technologies.
 
+## Documentation
+
+Further information can be found in the [documentation][documentation].
```

Comment:
Deepl is saying: "For more information, see the documentation." or "For more information, please refer to the documentation."